assets/source/setup-guide/app/steps/ClaimWebsite.js

```diff
@@ -35,47 +43,69 @@ const StaticError = ( { reqError } ) => {
 		return null;
 	}
 
+	const message = createInterpolateElement(
+		sprintf(
+			// translators: %s: error reason returned by Pinterest when verifying website claim fail.
+			__(
+				'<strong>We were unable to verify this domain.</strong> %s',
+				'pinterest-for-woocommerce'
+			),
+			reqError.message
+		),
+		{
+			strong: <strong />,
+		}
+	);
+
 	return (
-		<Text variant="body" className="claim-error">
-			{ reqError.message }
-		</Text>
+		<Notice status="error" isDismissible={ false }>
+			{ message }
+		</Notice>
 	);
 };
 
+/**
+ * Renders a UI with section block and <Card> to claim website (if not yet completed) and display its status.
+ *
+ * @param {Object} props React props.
+ * @param {'wizard'|'settings'} props.view Indicate which view this component is rendered on.
+ * @param {Function} [props.goToNextStep]
+ *   When the website claim is complete, called when clicking the "Continue" button.
+ *   The "Continue" button is only displayed when `props.view` is 'wizard'.
+ */
 const ClaimWebsite = ( { goToNextStep, view } ) => {
-	const [ status, setStatus ] = useState( 'idle' );
+	const [ status, setStatus ] = useState( STATUS.IDLE );
 	const [ reqError, setReqError ] = useState();
 	const isDomainVerified = useSettingsSelect( 'isDomainVerified' );
 	const setAppSettings = useSettingsDispatch( view === 'wizard' );
 	const createNotice = useCreateNotice();
+	const pfwSettings = wcSettings.pinterest_for_woocommerce;
 
 	useEffect( () => {
-		if ( status !== 'pending' && isDomainVerified ) {
-			setStatus( 'success' );
+		if ( status !== STATUS.PENDING && isDomainVerified ) {
+			setStatus( STATUS.SUCCESS );
 		}
 	}, [ status, isDomainVerified ] );
 
 	const handleClaimWebsite = async () => {
-		setStatus( 'pending' );
+		setStatus( STATUS.PENDING );
 		setReqError();
 
 		try {
 			const results = await apiFetch( {
-				path:
-					wcSettings.pinterest_for_woocommerce.apiRoute +
-					'/domain_verification',
+				path: pfwSettings.apiRoute + '/domain_verification',
 				method: 'POST',
 			} );
 
 			await setAppSettings( { account_data: results.account_data } );
 
-			setStatus( 'success' );
+			setStatus( STATUS.SUCCESS );
 		} catch ( error ) {
-			setStatus( 'error' );
+			setStatus( STATUS.ERROR );
 			setReqError( error );
 
 			createNotice(
-				'error',
+				STATUS.ERROR,
```

Comment:
💅  For this `createNotice`, I'd say we should use `'error'` instead of `STATUS.ERROR`. By using `STATUS.ERROR`, it gives a false impression that `createNotice` can work with any other `STATUS`.

```suggestion
				'error',
```

Reply:
Thanks for catching this! Yes, it shouldn't be changed. I changed it accidentally when I searched and replaced these hardcoded strings with editor. 🤦‍♂️